themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1435,7 +1439,14 @@ html body.tc-body.tc-single-tiddler-window {
 
 .tc-edit-field-add-value {
 	display: inline-block;
-	width: 35%;
+}
+
+@media (min-width: <<sidebarbreakpoint-plus-one>>) {
+
+	.tc-edit-field-add-value {
```

Comment:
what is the extra whitespace (new lines) needed for

Reply:
readability :grin: